CONTRIBUTING.md

```diff
@@ -76,6 +76,16 @@ when making a commit:
 pre-commit install
 ```
 
+We recommend [nextest](https://nexte.st/) to run Ruff's test suite (via `cargo nextest run`),
+though it's not strictly necessary:
+
+```shell
+cargo install nextest
+```
+
+Throughout this guide, any usages of `cargo test` can be replaced with `cargo nextest run`,
+if you choose to install `nextest`.
```

Comment:
We should add a note about `--failure-output immediate-final --no-fail-fast -j 12`

Reply:
Honestly tempted to omit `nextest` entirely since it just complicates things for new users.

Reply:
I would omit it, especially the part about recommending it because nextest doesn't run doctests (needs `cargo test --doc`), and needs to be installed